src/functionality.py

```diff
@@ -89,28 +880,121 @@ async def rpactive(ctx: commands.Context):
-            if suffix:
-                final_message += f'{suffix}'
+            other_timezones = str(irl_time)
+            other_timezones = other_timezones.split(" ")
+            hour_split = other_timezones[1].split(":")
+            hour_change = int(hour_split[0])
+            text_ = [
+                f"**-- Standard Time --**",
+                f"**EST**: {timezone_time_check(hour_change, -5)}:{hour_split[1]}:{hour_split[2]}",
+                f"**CST**: {timezone_time_check(hour_change, -6)}:{hour_split[1]}:{hour_split[2]}",
+                f"**MST**: {timezone_time_check(hour_change, -7)}:{hour_split[1]}:{hour_split[2]}",
+                f"**PST**: {timezone_time_check(hour_change, -8)}:{hour_split[1]}:{hour_split[2]}",
+                f"**-- Daylight Time --**",
+                f"**EDT**: {timezone_time_check(hour_change, -4)}:{hour_split[1]}:{hour_split[2]}",
+                f"**CDT**: {timezone_time_check(hour_change, -5)}:{hour_split[1]}:{hour_split[2]}",
+                f"**MDT**: {timezone_time_check(hour_change, -6)}:{hour_split[1]}:{hour_split[2]}",
+                f"**PDT**: {timezone_time_check(hour_change, -7)}:{hour_split[1]}:{hour_split[2]}",
+                f"**-- Europe Time --**",
+                f"**UTC-1**: {timezone_time_check(hour_change, -1)}:{hour_split[1]}:{hour_split[2]}",
+                f"**UTC**: {timezone_time_check(hour_change, 0)}:{hour_split[1]}:{hour_split[2]}",
+                f"**UTC+1**: {timezone_time_check(hour_change, 1)}:{hour_split[1]}:{hour_split[2]}",
+                f"**UTC+2**: {timezone_time_check(hour_change, 2)}:{hour_split[1]}:{hour_split[2]}",
+                f"**UTC+3**: {timezone_time_check(hour_change, 3)}:{hour_split[1]}:{hour_split[2]}",
+            ]
 
-            await message.delete()
-            await to_channel.send(final_message)
+            text_ = "\n".join(text_)
+
+            embed = discord.Embed(title=title_embed,
+                                  description=f'{text_}',
+                                  colour=discord.Color.dark_gold())
+            embed.set_thumbnail(url=ctx.author.avatar_url)
+            embed.set_footer(text=ctx.author)
+
+            await ctx.send(embed=embed)
+
+        @bot.command(
+            name='utc',
+            brief='Lists the time for UTC timezones',
+            help=('Lists the time for UTC timezones. There is an optional argument; if you input an epoch/unix time after the command, '
+                  'you will get said time instead. Date format: YYYY-MM-DD.'
+                  '\nArguments: $utc <seconds>'
+                  '\nExample: $utc 1'
+                  '\n\n<seconds> is an optional field; it is an epoch/unix second argument and MUST be an integer.')
+        )
+        async def utc(ctx: commands.Context, seconds: int = None):
+            if not _validate_command(ctx):
+                return
+
+            irl_time = time.strftime('%Y-%m-%d %H:%M:%S', time.gmtime(seconds))
```

Comment:
This produces a silent TypeError if `seconds` is not a number. Add an explicit error for that case.